mesonbuild/backend/ninjabackend.py

```diff
@@ -2360,7 +2360,7 @@ def generate_custom_target_clean(self, outfile, trees):
         e.add_item('COMMAND', [sys.executable,
                                self.environment.get_build_command(),
                                '--internal', 'cleantrees', d_file])
-        e.add_item('description', 'Cleaning CustomTarget directories')
+        e.add_item('description', 'Cleaning custom targets directories.')
```

Comment:
Whether it's a single custom target or multiple, the pluralization of "directories" covers it (especially if there were only one directory per custom target, but I don't know if that's the case.) The double-plurals just looks odd. It could be fixed by using a possessive (custom targets' directories) or swapping the order (directories of custom targets) or dropping the first plural.